sway/commands/fullscreen.c

```diff
@@ -0,0 +1,40 @@
+#include <wlr/types/wlr_wl_shell.h>
+#include "log.h"
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/tree/container.h"
+#include "sway/tree/view.h"
+#include "sway/tree/layout.h"
+
+// fullscreen toggle|enable|disable
+struct cmd_results *cmd_fullscreen(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if (config->reading) return cmd_results_new(CMD_FAILURE, "fullscreen", "Can't be used in config file.");
+	if (!config->active) return cmd_results_new(CMD_FAILURE, "fullscreen", "Can only be used when sway is running.");
```

Comment:
Braces are mandatory and these reading/active checks are now done at commands.c by placing it in the appropriate command block